content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -37,73 +37,89 @@ further_reading:
+- Enable the Continuous Profiler, ingesting 100% of traces, and Trace ID injection into logs during setup.
 
-Otherwise, to begin tracing applications written in any language, first [install and configure the Datadog Agent][3]. The .NET Tracer runs in-process to instrument your applications and sends traces from your application to the Agent.
+### Automatic Instrumentation
 
-**Note**: The .NET Tracer supports all .NET-based languages (C#, F#, Visual Basic, etc).
+<div class="alert alert-warning" style="font-style: italic;"> 
+  <p style="font-weight: bold; margin-bottom: 0px;">Note:<p/> 
+    When using both custom and automatic instrumentation, it is important to keep the MSI installer and NuGet package versions in sync.
+</div>
 
-## Automatic Instrumentation
+Otherwise, to begin tracing .NET applications: 
 
-Automatic instrumentation can collect performance data about your application with zero code changes and minimal configuration. The .NET Tracer automatically instruments all [supported libraries][1] out of the box.
+#### IIS Applications 
 
-Automatic instrumentation captures:
+1. Install and configure the [Windows Datadog Agent][2]. 
 
-- Execution time of instrumented calls
-- Relevant trace data, such as URL and status response codes for web requests or SQL queries for database access
-- Unhandled exceptions, including stacktraces if available
-- A total count of traces (e.g. web requests) flowing through the system
+2. Download the .NET Tracer [MSI installer][3].
 
-### Installation
+3. Select the MSI installer for the architecture that matches the operating system (x64 or x86).
 
-To use automatic instrumentation on Windows, install the .NET Tracer on the host by running the [MSI installer for Windows][4] as administrator. Choose the installer for the architecture that matches the operating system (x64 or x86).
+4. Install the .NET Tracer MSI installer with administrator privileges.
 
-After installing the .NET Tracer, restart applications so they can read the new environment variables. To restart IIS, run the following commands as administrator:
+5. Restart IIS using the following command within an a: 
 
-```cmd
-net stop /y was
-net start w3svc
-```
+   ```text
+   net stop /y was
+   net start w3svc
+   ```
+6. Create application load. 
+
+7. Visit Datadog's [APM Live Traces][4]. 
 
-### Required Environment Variables
+### Required Environment Variables 
 
-If your application runs in IIS, skip the rest of this section.
+<div class="alert alert-warning" style="font-style: italic;"> 
+  <p style="font-weight: bold; margin-bottom: 0px;">Note:<p/> The .NET runtime tries to load a profiler into any .NET process started with these environment variables set. You should limit instrumentation only to the applications that need to be traced. 
+  <p style="font-weight: bold; margin-bottom: 0px">We do not recommend setting these environment variables globally as this causes all .NET processes on the host to load the profiler.<p/>
+</div>
```

Comment:
We've jumped up to an h3 here, and it's unclear to me if these notes apply to the IIS app tracing instructions or the non-IIS app tracing instructions, or both. I *think* it's referring to the env vars mentioned below, and if so, we should put this info right beside them, not separated by the "Non-IIS Applications" heading, and not introduced by an h3. (That is, I think the docs structure for this was better the way it was, although we can add the alert-warning box around it if that helps.)

Reply:
I moved this around a bit. let me know what you think.